### DISCUSSION TITLE
Where can i find my personal SA-Coordinator's name and email??

### DISCUSSION THREAD

**Discussion by Hardik0803** [microsoft/SATechnicalOnboarding#114] [Questions]
Where can i find my personal SA-Coordinator's name and email??

**Comment by AnthonyByansi** [comment]
Hi @Hardik0803. If you are on teams, please utilize the search bar at the top of any chat and type `SA-coordinator`. will see his profile in a blue icon. That is the person you need to talk to. I would have provided the email here but that is unethical to provide publicly.
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/411b5a03-e6f6-4206-8563-dbf9d7c9ed81)


Alternatively, you can find the support team details at our hand-book in the [SA-Help section](https://stdntpartners.sharepoint.com/sites/SAProgramHandbook/SitePages/Program-Directory.aspx)
